dart/lib/src/open_location_code.dart

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
_This is an automated bot comment from the TravisCI tests_  
**File has `dartfmt` errors that must be fixed**. Here is a diff, or run `format_check.sh`:
<pre>19c19  
<    import 'dart:math';  
---  
> import 'dart:math';</pre>